src/main/java/com/yahoo/bullet/dsl/BulletDSLConfig.java

```diff
@@ -192,4 +252,39 @@ public BulletConfig validate() {
         VALIDATOR.validate(this);
         return this;
     }
+
+    private static Object readFile(Object file) {
+        String fileName = (String) file;
+        if (fileName.startsWith(FILE_PREFIX)) {
+            fileName = fileName.substring(FILE_PREFIX.length());
+            try {
+                InputStream is = BulletDSLConfig.class.getResourceAsStream("/" + fileName);
+                if (is != null) {
+                    return writeToString(is);
+                }
+                return writeToString(new FileInputStream(new File(fileName)));
+            } catch (IOException e) {
+                throw new RuntimeException("Could not read file: " + fileName, e);
+            }
+        }
+        return file;
+    }
+
+    private static String writeToString(InputStream is) throws IOException {
+        ByteArrayOutputStream output = new ByteArrayOutputStream();
```

Comment:
Can use something simpler since it's just being read to a string: 
```new BufferedReader(new InputStreamReader(is)).lines().collect(Collectors.joining("\n"));```?